src/App.tsx

```diff
@@ -1,14 +1,52 @@
-import React from 'react';
+import React, { useMemo, useRef, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [isInput, setIsInput] = useState('');
```

Comment:
when you name state or var starting with `is` we expect that it will be the boolean var instead of string, so let's rename it